packages/drivers/routerlicious-driver/src/wholeSummaryUploadManager.ts

```diff
@@ -0,0 +1,64 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import {
```

Comment:
curious, why do we need to clone it?

Reply:
Because it takes gitmanager in constructor which we have changed, and it became incompatible with older wholeSummaryUploadManager.